### PR TITLE
feat(redocly): check bundle and run lint

### PR DESCRIPTION
**Pull request by MathieuVeber**

### CODE REVIEW

File: .github/workflows/redocly.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+            - name: Install Dependencies
+              run: npm ci
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.READONLY_NPM_TOKEN }}
+
+            - name: Run Redocly Lint
+              run: npm run redocly:lint --if-present
+
+            - name: Is the bundle up to date?
```
Comment (Styrke) [inline .github/workflows/redocly.yml:36]: I don't think questions are good as names for workflow steps.

How about "Check that bundle is up to date"?

**Review by Styrke — CHANGES_REQUESTED** [review]

File: .github/workflows/redocly.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                required: true
+
+jobs:
+    redocly:
+        runs-on: ubuntu-latest
+        timeout-minutes: 1
+
+        steps:
+            - name: Checkout Repository
+              uses: actions/checkout@v4
```
Comment (aimeerivers) [inline .github/workflows/redocly.yml:17]: maybe add `with: fetch-depth: 1` so it doesn't clone the entire repository

**Review by aimeerivers — CHANGES_REQUESTED** [review]
just a small suggestion.

File: .github/workflows/redocly.yml (hunk 2 above)
Comment (MathieuVeber) [reply .github/workflows/redocly.yml:17]: It's the default https://github.com/actions/checkout?tab=readme-ov-file#usage

**Comment by aimeerivers** [comment]
do we have an example of a PR that runs these checks (and fails?) .. would be good to see how it looks.

File: .github/workflows/redocly.yml (hunk 2 above)
Comment (aimeerivers) [reply .github/workflows/redocly.yml:17]: 😮

**Comment by MathieuVeber** [comment]
> do we have an example of a PR that runs these checks (and fails?) .. would be good to see how it looks.

Yes in [data-service](https://github.com/hedia-team/data-service/pull/38)

Check out the failing actions 😉 https://github.com/hedia-team/data-service/actions/workflows/continuous-integration.yml

File: .github/workflows/redocly.yml (hunk 2 above)
Comment (aimeerivers) [reply .github/workflows/redocly.yml:17]: so it is!! i never knew!

**Review by aimeerivers — APPROVED** [review]
excellent! i really like your step naming too! was going to say something about imperative verbs, but i see you already fixed it! 😄

**Review by aimeerivers — APPROVED** [review]
just coming back to confirm, i don't require any changes.

**Comment by MathieuVeber** [comment]
> just coming back to confirm, i don't require any changes.

This is not about you it's about @Styrke 😜

**Review by Styrke — APPROVED** [review]
Looks good. Looking forward to it. 👍

Sorry, I didn't realize the PR was waiting for me.

**Pull request merged**